lib/reline/line_editor.rb

```diff
@@ -1654,6 +1654,12 @@ def finish
   end
   alias_method :next_history, :ed_next_history
 
+  private def end_of_history(key)
```

Comment:
I'm not sure of the naming convention. Should this be named `ed_end_of_history`? It seem like Editline supports that.

Reply:
`ed_end_of_history` looks good :+1:. And we need to alias it to `end_of_history`